docs/CUSTOM_RULES_101.md

```diff
@@ -486,14 +476,14 @@ class MyJavaFileCheckRegistrarTest {
 
 At this point, we've completed the implementation of a first custom rule and registered it into the custom plugin. The last remaining step is to test it directly with the SonarQube platform and try to analyze a project! 
 
-Start by building the project using maven. Note that here we are using the self-contained `pom` file targeting SonarQube `8.7`. If you renamed it into `pom.xml`, remove the `-f pom_SQ_8_7.xml` part of the following command):
+Start by building the project using maven. Note that here we are using the self-contained `pom` file targeting SonarQube `8.8`. If you renamed it into `pom.xml`, remove the `-f pom_SQ_8_8.xml` part of the following command):
 
 
 ```
 $ pwd
 /home/gandalf/workspace/sonar-java/docs/java-custom-rules-example
   
-$ mvn clean install -f pom_SQ_8.7.xml
+$ mvn clean install -f pom_SQ_8.8.xml
```

Comment:
Isn't there a step missing here about generating the custom rule's metadata files?

Reply:
nop, if you use all the parameters of the annotation, you don't need the metadata files I think